corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -302,6 +296,7 @@ def test_excluded_case_types_are_not_forwarded(self):
         CaseFactory(self.domain_name).post_case_blocks([non_white_listed_case])
         self.assertEqual(1, len(self.repeat_records(self.domain_name).all()))
 
+    # @run_with_all_backends
```

Comment:
@sravfeyn i think you were the first to implement this spec, do you think getting the user id of the form that submitted the case block is equivalent to the user id of the case block?